format/Layout.md

```diff
@@ -583,6 +583,12 @@ even if the null bitmap of the parent union array indicates the slot is
 null.  Additionally, a child array may have a non-null slot even if
 the the types array indicates that a slot contains a different type at the index.
 
+## Dictionary encoding
+
+When a field is dictionary encoded, the values are represented by an array of Int32 representing the index of the value in the dictionary.
+The Dictionary is received as a DIctionaryBacth whose id is referenced byt the dictionary field in the Field.
```

Comment:
Example will definitely help to illustrate. Notably in an IPC / RPC setting when you encounter a dictionary-encoded field you can "move on" because the data at that level of the hierarchy is integers referencing a dictionary that you presumably will have received in a separate message